### PR TITLE
[move-examples] changed option<u64> to u64 and added big_vec and buck…

### PR DESCRIPTION
**Pull request by 0xchloe**
…et_table as part of the package

### CODE REVIEW

**Review by movekevin — APPROVED** [review]

**Comment by lightmark** [comment]
Could you hold on as I am changing big_vector code?

File: aptos-move/move-examples/post_mint_reveal_nft/Move.toml [hunk 1]
```diff
@@ -9,4 +9,4 @@ AptosToken = { local = "../../framework/aptos-token" }
 DataStructure = { local = "../data_structures" }
 
 [addresses]
-source_addr = "0xcafe"
+source_addr = "_"
```
Comment (lightmark) [inline aptos-move/move-examples/post_mint_reveal_nft/Move.toml:12]: This is intentional?

Comment (gregnazario) [reply aptos-move/move-examples/post_mint_reveal_nft/Move.toml:12]: This should be handled in this PR https://github.com/aptos-labs/aptos-core/pull/5955

Comment (movekevin) [reply aptos-move/move-examples/post_mint_reveal_nft/Move.toml:12]: Yes https://github.com/aptos-labs/aptos-core/pull/5955/files/cd5196cc860e0d3646a1058218703b28a8b1fd15..bcd4a50a202504073d0748590ea09b63f488fd6e has more context

Comment (lightmark) [reply aptos-move/move-examples/post_mint_reveal_nft/Move.toml:12]: Then please rebase on PR https://github.com/aptos-labs/aptos-core/pull/5955. Thanks for clarifying

**Review by lightmark — APPROVED** [review]

**Pull request merged**